app/controllers/posts_controller.rb

```diff
@@ -3,10 +3,31 @@ def index
     @posts = Post.all
     @user = User.where(id: params[:user_id]).first
     @comments = Comment.all
+    @like = Like.all
   end
 
   def show
     @post = Post.find(params[:id])
     @comments = Comment.all
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    puts params
+    user = current_user
+    new_post = Post.new(post_params)
+    new_post.author = user
+    if new_post.save
+      redirect_to user_posts_path(id: user.id)
+    else
+      redirect_to user_posts_path(user_id: user.id)
+    end
+  end
+
+  def post_params
+    params.require(:post).permit(:Title, :Text)
+  end
```

Comment:
- [ ] I tried creating a post but no post was created.

![Screenshot from 2023-02-03 21-43-23](https://user-images.githubusercontent.com/26767383/216652829-800552cc-023c-4f51-8dc8-4033338c8609.png)

### After submitting the form, There is no post. Please fix this.

![Screenshot from 2023-02-03 21-42-16](https://user-images.githubusercontent.com/26767383/216652928-d287b41c-0a78-4c34-9273-f20522eb75df.png)